communitybsl.properties

```diff
@@ -1,14 +1,20 @@
 category=Languages
 description=Code Analyzer for 1C (BSL)
 homepageUrl=https://1c-syntax.github.io/sonar-bsl-plugin-community/en
-archivedVersions=1.9.0,1.10.0,1.11.0
-publicVersions=1.12.0
+archivedVersions=1.9.0,1.10.0,1.11.0,1.12.0
+publicVersions=1.13.0
 
 defaults.mavenGroupId=com.github.1c-syntax
 defaults.mavenArtifactId=sonar-communitybsl-plugin
 
+1.13.0.description=Speed optimization
+1.13.0.sqVersions=[9.9,LATEST]
+1.13.0.date=2023-07-31
+1.13.0.changelogUrl=https://github.com/1c-syntax/sonar-bsl-plugin-community/releases/tag/v1.13.0
+1.13.0.downloadUrl=https://github.com/1c-syntax/sonar-bsl-plugin-community/releases/download/v1.13.0/sonar-communitybsl-plugin-1.13.0.jar
+
 1.12.0.description=9 new rules, bug fixes, optimize
-1.12.0.sqVersions=[9.9,LATEST]
+1.12.0.sqVersions=[9.9,9.9.*]
```

Comment:
This range needs to end with the current version. Otherwise people on 10.1 won't be prompted to upgrade.

Reply:
fixed!